doc/development/contributing.rst

```diff
@@ -304,11 +304,11 @@ be reflected the next time you open a Python interpreter and ``import mne``
 Finally, we'll add a few dependencies that are not needed for running
 MNE-Python, but are needed for locally running our test suite::
 
-    $ pip install -e ".[test]"
+    $ pip install --group=test
```

Comment:
I don't think we need the `.` here anymore, as we already `pip install -e .`ed above; now, `pip install --group=…` should only install the dependency group in addition to what's already in the venv